library_generation/test/integration_tests.py

```diff
@@ -12,103 +12,84 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import json
+from filecmp import cmp
+from filecmp import dircmp
+from git import Repo
 import os
 import shutil
+import subprocess
 import unittest
 from distutils.dir_util import copy_tree
 from distutils.file_util import copy_file
-from filecmp import cmp
-from filecmp import dircmp
-
-from git import Repo
 from pathlib import Path
-from typing import List
-
-from library_generation.generate_pr_description import generate_pr_descriptions
-from library_generation.generate_repo import generate_from_yaml
-from library_generation.model.generation_config import from_yaml, GenerationConfig
+from library_generation.model.generation_config import GenerationConfig
+from library_generation.model.generation_config import from_yaml
 from library_generation.test.compare_poms import compare_xml
-from library_generation.utils.utilities import (
-    sh_util as shell_call,
-    run_process_and_print_output,
-)
+from library_generation.utils.utilities import sh_util as shell_call
 
-config_name = "generation_config.yaml"
 script_dir = os.path.dirname(os.path.realpath(__file__))
-# for simplicity, the configuration files should be in a relative directory
-# within config_dir named {repo}/generation_config.yaml, where repo is
-# the name of the repository the target libraries live.
-config_dir = f"{script_dir}/resources/integration"
-golden_dir = f"{config_dir}/golden"
+golden_dir = os.path.join(script_dir, "resources", "integration", "golden")
+repo_root_dir = os.path.join(script_dir, "..", "..")
+build_file = os.path.join(
+    repo_root_dir, ".cloudbuild", "library_generation", "library_generation.Dockerfile"
+)
+image_tag = f"test-image:latest"
```

Comment:
We don't have to use f string as this is a hardcoded value.

Reply:
Done